package-drone/src/main/java/de/dentrassi/pm/jenkins/AbstractUploader.java

```diff
@@ -44,12 +37,19 @@ public AbstractUploader ( final Run<?, ?> run )
 
     protected void fillProperties ( final Map<String, String> properties )
     {
-        final String jenkinsUrl = Jenkins.getInstance ().getRootUrl ();
-        if ( jenkinsUrl != null )
-        {
-            final String url = jenkinsUrl + this.run.getUrl ();
-            properties.put ( "jenkins:buildUrl", url );
-        }
+    	final Jenkins instance = Jenkins.getInstance();
+    	if (instance != null)
+    	{
+	        final String jenkinsUrl = instance.getRootUrl ();
+	        if ( jenkinsUrl != null )
+	        {
+	            final String url = jenkinsUrl + this.run.getUrl ();
+	            properties.put ( "jenkins:buildUrl", url );
+	        }
+	   	}
+
+        final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat ( "yyyy-MM-dd HH:mm:ss.SSS" );
```

Comment:
In java the name of local variable must be lowercase.
Anyway to avoid to pay instantiation of this formatter each call of fillProperties you could instantiate it in the constructor.